module/enrichers.mjs

```diff
@@ -636,6 +637,95 @@ async function enrichSave(config, label, options) {
 
 /* -------------------------------------------- */
 
+/**
+ * Enrich an item use link to roll an item on the selected token.
+ * @param {string[]} config            Configuration data.
+ * @param {string} [label]             Optional label to replace default text.
+ * @returns {Promise<HTMLElement|null>}         An HTML link if the item link could be built, otherwise null.
```

Comment:
```suggestion
 * @param {string[]} config              Configuration data.
 * @param {string} [label]               Optional label to replace default text.
 * @returns {Promise<HTMLElement|null>}  An HTML link if the item link could be built, otherwise null.
```